xml/docu_styleguide.outline.xml

```diff
@@ -91,7 +91,7 @@
        <formalpara>
         <title>Authorship Information</title>
         <para>
-         Create a separate file <filename>authors.xml</filename> and add an
+         If authors should be listed for a book, create a separate file <filename>authors.xml</filename> and add an
```

Comment:
Hum, right. I guess if we're making it officially optional, we should probably have some kind of guidance when to include authors and when not to..?

Reply:
For SLE, after a long discussion, we decided to keep a list of authors. However, it is still undecided in which form (probably not in an authors.xml file any more). As this might differ from product to product, I would therefore suggest to remove the para about authors from the style guide.

Reply:
Done in the upcoming commit.

Reply:
I have no issue with removing authorship information per se. But I think if we're removing this, we should have a decision about it from the entire Stylewich team.

I'd suggest a rule like the following, to accommodate both the regular docs and the SBPs:
* for regular documentation: do not include authorship information
* for articles where the main contributor is not part of the documentation team, consider including authorship information, in accordance with the author's wishes

Reply:
Seeing as we discussed this today and agreed upon a different solution, I guess this now needs to changed (again).

Reply:
This is obsolete meanwhile, please see https://github.com/SUSE/doc-styleguide/pull/155